src/components/SearchForm/SearchForm.spec.tsx

```diff
@@ -0,0 +1,97 @@
+describe('Search Form', () => {
+    it(`Search Form: Should render form`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`form`).should('exist')
+    })
+
+    it(`Search Form: Should render c-searchform`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`.c-searchform`).should('exist')
+    })
+
+    it(`Search Form: Should render label`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`label`).should('exist')
+    })
+
+    it(`Search Form: Should render input`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`input`).should('exist')
+    })
+
+    it(`Search Form: Should render button`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`button`).should('exist')
+    })
+
+    it(`Search Form: Should render svg`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`svg`).should('exist')
+    })
+
+    it(`Search Form: Should render span`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`span`).should('exist')
+    })
+
+    it(`Search Form: Should render u-visually-hidden`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get(`.u-visually-hidden`).should('exist')
+    })
+
+    it(`Search Form: Should render submit`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-search-form--base`
+        )
+        cy.get('form').submit().should('exist')
+        cy.on('window:alert',(txt)=>{
+            expect(txt).to.contains('test');
+         })
+    })
+
+    it(`Search Form: Should render action`, () => {
```

Comment:
Great job!
Two small things:
1. Change the title to 'test' instead of 'render'
2. Add another test for the click itself